client/connect-account-page/index.tsx

```diff
@@ -192,7 +264,24 @@ const ConnectAccountPage: React.FC = () => {
 							{ strings.nonSupportedCountry }
 						</BannerNotice>
 					) }
-					{ devMode && <SandboxModeNotice /> }
+					{
+						// Show general sandbox notice when no account is connected but sandbox mode is active.
+						! isAccountConnected && devMode ? (
+							<SandboxModeNotice />
+						) : (
+							// If we already have a sandbox account connected (but in a bad state) and
```

Comment:
what means `but in a bad state`?

Reply:
It was missing `! isAccountValid &&` but that is implied since you are on the Connect page with a connected account. I will add it for clarity.